core/utils_test.py

```diff
@@ -66,7 +66,7 @@ def test_yaml_dict_conversion(self) -> None:
         test_dicts = [{}, {'a': 'b'}, {'a': 2}, {'a': ['b', 2, {'c': 3.5}]}]
 
         for adict in test_dicts:
-            yaml_str = python_utils.yaml_from_dict(adict) # type: ignore[no-untyped-call]
+            yaml_str = utils.yaml_from_dict(adict)  # type: ignore
```

Comment:
```suggestion
            yaml_str = utils.yaml_from_dict(adict)
```

Reply:
this was done because I was getting this error in mypy checks
core/utils_test.py:69: error: Argument 1 to "yaml_from_dict" has incompatible type "object"; expected "Dict[str, Any]"  [arg-type]
core/utils_test.py:78: error: Incompatible types in assignment (expression has type "Dict[str, Any]", variable has type "str")  [assignment]

Reply:
can u identify why its throwing that error.
it may be because of that empty dictionary in your test case

Reply:
The tests seem to have passed still mentioning it , it was because of these two lines 
https://github.com/oppia/oppia/blob/8b1986ef0746ebec1d52c3243c078cc58bdd15a0/core/utils_test.py#L69

and this line 
https://github.com/oppia/oppia/blob/8b1986ef0746ebec1d52c3243c078cc58bdd15a0/core/utils_test.py#L78

Reply:
Can you try to remove the ignore and the empty dict from list on line 66.

Reply:
okay checking it

Reply:
same error 
core/utils_test.py:69: error: Argument 1 to "yaml_from_dict" has incompatible type "object"; expected "Dict[str, Any]"  [arg-type]
Found 1 error in 1 file (checked 1 source file)

![image](https://user-images.githubusercontent.com/45638240/143775667-93ad347c-3dbe-4b66-ae4e-5011df15b8e1.png)

Reply:
```suggestion
        test_dicts:List[Dict[str, Any]] = [
          {}, {'a': 'b'}, {'a': 2}, {'a': ['b', 2, {'c': 3.5}]}]
```
try this, mypy is unable to infer the type here so you will have to explicitly specify it

Reply:
okay adding it

Reply:
yes fixed